libs/server/Resp/Objects/ObjectStoreUtils.cs

```diff
@@ -65,5 +65,38 @@ private bool AbortWithErrorMessage(int count, ReadOnlySpan<byte> errorMessage)
 
             return true;
         }
+
+        /// <summary>
+        /// Tries to parse the input as "LEFT" or "RIGHT" and returns the corresponding OperationDirection.
+        /// If parsing fails, returns OperationDirection.Unknown.
+        /// </summary>
+        /// <param name="input">The input to parse.</param>
+        /// <returns>The parsed OperationDirection, or OperationDirection.Unknown if parsing fails.</returns>
+        public OperationDirection GetOperationDirection(ArgSlice input)
+        {
+#if NET8_0_OR_GREATER
```

Comment:
there is no need for the conditional now. just use the else part for all .NET versions.

Reply:
Good suggestion, I have fixed it, thank you for your review again!